presto-mixin/dashboards/presto-overview.libsonnet

```diff
@@ -0,0 +1,962 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'presto-overview';
+
+local promDatasourceName = 'prometheus_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local activeResourceManagersPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'com_facebook_presto_metadata_DiscoveryNodeManager_ActiveResourceManagerCount{job=~"$job", presto_cluster=~"$presto_cluster"}',
+      datasource=promDatasource,
+      legendFormat='{{presto_cluster}}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Active resource managers',
```

Comment:
I'm not sure it would make sense in this context. The amount of resource managers is completely dependent on the type of presto environment, the thresholds of red/green wouldn't really hold too well. In the case of the coordinator, i can see having 0 as being red and anything over being green, but then that as a gauge seems out of place with the resource manager being a stat. The same can be done for workers with 0 for red and green for anything over, but then again, it seems overkill and out of place as a gauge. Feel free to push back on this, but how about this: i keep these as stat panels and add colors to number on the stat depending on what it is.

Reply:
Yep that makes sense, thanks for explaining!
Let's stick to coloured stat panels then, though I might still suggest dropping the graph background